eng/CodeQL.ps1

```diff
@@ -0,0 +1,195 @@
+function Install-Gdn {
+    param(
+        [string]$Path
+    )
+
+    Start-Process nuget -Verbose -ArgumentList "install", "Microsoft.Guardian.Cli", "-Source https://securitytools.pkgs.visualstudio.com/_packaging/Guardian/nuget/v3/index.json", "-OutputDirectory $Path", "-NonInteractive", "-NoCache" -NoNewWindow -Wait
```

Comment:
I just realized here you've implemented version floating for guardian CLI; eng/common very much needs this.

Reply:
Yep. That's dotnet/core-eng#14160 and I think this is the approach I'll take.